source/_integrations/google_tasks.markdown

```diff
@@ -0,0 +1,73 @@
+---
+title: Google Tasks
+description: Instructions on how to use Google Tasks in Home Assistant.
+ha_category:
+  - To-do List
+ha_iot_class: Cloud Polling
+ha_release: 2023.11
+ha_config_flow: true
+ha_domain: google_tasks
+ha_platforms:
+  - todo
+ha_codeowners:
+  - '@allenporter'
+ha_integration_type: integration
+---
+
+The Google Tasks integration allows you to connect your [Google Tasks](https://support.google.com/tasks/answer/7675772) to Home Assistant. The integration adds a [To-do List entity](/integrations/todo) for
+each Task list allowing you create, update, or delete items on the list from the To-do List dashboard.
+
+## Prerequisites
+
+You need to configure developer credentials to allow Home Assistant to access your Google Account.
+These credentials are the same as the ones for [Nest](/integrations/nest) and [Google Mail](/integrations/google_mail).
+These are not the same as the one for [Google Calendar](/integrations/google).
+If you have already set up the correct credentials, you can do step 1 and then skip to step 13 on the below instructions.
+
+{% details "Generate Client ID and Client Secret" %}
+
+This section explains how to generate a Client ID and Client Secret on
+[Google Developers Console](https://console.cloud.google.com/apis/library/tasks.googleapis.com).
+
+1. First, go to the Google Developers Console to enable [Google Tasks](https://console.cloud.google.com/apis/library/tasks.googleapis.com)
+2. The wizard will ask you to choose a project to manage your application. Select a project and select **Continue**.
+3. Verify that your Google Tasks API was enabled and select **Go to credentials**.
+4. Navigate to **APIs & Services** (left sidebar) > [Credentials](https://console.cloud.google.com/apis/credentials)
+5. Click on the field on the left of the screen, **OAuth Consent Screen**.
+6. Select **External** and **Create**.
+7. Set the **App Name** (the name of the application asking for consent) to anything you want, e.g., *Home Assistant*.
+8. You then need to select a **Support email**. To do this, from the dropdown menu, select your email address.
+9. You finally need to complete the section: **Developer contact information**. To do this, enter your email address (the same as above is fine).
+10. Scroll to the bottom and select **Save and Continue**. You don't have to fill out anything else, or it may enable additional review.
+11. You will then be automatically taken to the **Scopes** page. You do not need to add any scopes here, so select **Save and Continue** to move to the **Optional info** page. You do not need to add anything to the **Optional info** page, so select **Save and Continue**, which will take you to the **Summary** page. Select **Back to Dashboard**.
+12. Select **OAuth consent screen** again and set *Publish Status* to **Production**. Otherwise your credentials will expire every 7 days.
```

Comment:
```suggestion
12. Select **OAuth consent screen** again and set **Publish Status** to **Production**. Otherwise, your credentials will expire every 7 days.
```